mappings/net/minecraft/server/player/ServerPlayer.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_69 net/minecraft/server/player/ServerPlayer
+	FIELD field_255 packetHandler Lnet/minecraft/class_11;
+	FIELD field_256 server Lnet/minecraft/server/MinecraftServer;
+	FIELD field_259 visibleItems [Lnet/minecraft/class_31;
+	FIELD field_261 interactionManager Lnet/minecraft/class_70;
+	FIELD field_264 trackedChunkList Ljava/util/List;
+	FIELD field_265 trackedChunkSet Ljava/util/Set;
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;Lnet/minecraft/class_18;Ljava/lang/String;Lnet/minecraft/class_70;)V
+		ARG 1 server
+		ARG 2 level
+		ARG 3 name
+		ARG 4 interactionManager
+	METHOD method_307 onContainerClosed ()V
+	METHOD method_310 refreshPose (FFZZFF)V
```

Comment:
why refresh

Reply:
is it a setter?

Reply:
if it's a setter then setPose

Reply:
it’s refreshPosirionAndAngles in yarn

Reply:
refresh because it sets every single thing